lib/ace/ace.js

```diff
@@ -55,6 +55,7 @@ require("./multi_select");
 require("./mode/folding/fold_mode");
 require("./theme/textmate");
 require("./ext/error_marker");
+require("./lib/katex/katex")
```

Comment:
No, it is necessary for KaTeX to actually be included in the project. Otherwise you can't `ace.require("ace/lib/katex/katex")`.